sympy/solvers/ode/hypergeometric.py

```diff
@@ -169,11 +177,11 @@ def match_2nd_2F1_hypergeometric(I, k, sing_point, func):
     I = I.subs(x, t)
     I = I*(t.diff(x))**2
     I = factor(I)
-    dict_I = {x**2:0, x:0, 1:0}
+    dict_I = {x**2: 0, x: 0, 1: 0}
     I0_num, I0_dem = I0.as_numer_denom()
     # collecting coeff of (x**2, x), of the standerd equation.
     # substituting (a-b) = s, (a+b) = r
-    dict_I0 = {x**2:s**2 - 1, x:(2*(1-r)*c + (r+s)*(r-s)), 1:c*(c-2)}
+    dict_I0 = {x**2: s**2 - 1, x: (2*(1-r)*c + (r+s)*(r-s)), 1: c*(c-2)}
```

Comment:
it is reluctant to make changes in values of the dict; compare this to the addition of space arount the `+` at line 198 a few lines below

Reply:
I think this is due to this option:
https://github.com/sympy/sympy/blob/2d90dc1d91b1d16679bec2c0f94d49b836b5315b/setup.cfg#L95

If you remove that option, it becomes:
```python
    dict_I0 = {
        x**2: s**2 - 1,
        x: (2*(1 - r)*c + (r + s)*(r - s)),
        1: c*(c - 2)
    }
```

Reply:
(line-breaks are just because this didn't fit within 79 anymore)

Reply:
that looks better

Reply:
Agreed. I switched it off (ba8564764b), but we have to check if that has undesired consequences elsewhere.